src/graphics/threejs_export.hpp

```diff
@@ -30,6 +32,8 @@ class Threejs_export
 	int morphVertices, morphColours, morphNormals;
 	int number_of_time_steps;
 	char *groupName, *regionPath;
+	//This is a weak pointer
+	cmzn_graphics_id graphics;
```

Comment:
Not actually a weak pointer; in internal code I no longer use the _id, so I would put:
```
	cmzn_graphics *graphics;  // not accessed
```
Replace cmzn_graphics_id with cnzn_graphics * in other new code in this PR.
Note the only place I use the _id in internal code is in the implementation of a public API otherwise VS doesn't recognize it as such.